src/components/Banner/alert_templates.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import { makeGitHubIssueLink } from "../../colorizer/utils/url_utils";
+import { VERSION_NUMBER } from "../../constants";
+
+import { AlertBannerProps } from "./AlertBanner";
+
+const MAX_ERROR_MESSAGE_LENGTH = 50;
+
+/** Trims whitespace at the start and end of each line in a multiline string. */
+const trimMultilineString = (str: string): string => {
+  return str
+    .split("\n")
+    .map((line) => line.trim())
+    .join("\n");
+};
+
+/**
+ * Shows an alert banner for when viewer settings could not be loaded
+ * from the URL. Includes a link to report the issue on GitHub.
+ */
+export const showFailedUrlParseAlert = (url: string, error: Error): AlertBannerProps => {
+  let errorMessage = error.message;
+  if (errorMessage.length > MAX_ERROR_MESSAGE_LENGTH) {
+    errorMessage = errorMessage.slice(0, MAX_ERROR_MESSAGE_LENGTH) + "...";
+  }
+  // Break URL into lines for each param for better readability
+  url = url.replace("?", "?\n");
+  url = url.replaceAll("&", "\n&");
```

Comment:
Just curious: is this string replacement robust enough for all possible input url formats?

Reply:
It shouldn't cause any problems if the URL does, I think, since we're only adding line breaks.
For example, if you copy the below into the address bar, it should still work as normal.
```
https://timelapse.allencell
.org
```